acceptance/suites/pre_suite/foss/10_install_release_repos.rb

```diff
@@ -1,5 +1,8 @@
 step "Setup Puppet Labs Release repositories." do
   hosts.each do |host|
-    install_puppetlabs_release_repo host
+    platform = host.platform
```

Comment:
Would it be possible to use `confine :to, :platform => 'windows'`? I don't actually know how that works/where it can be used, but this does seem like the place for it, if possible.